examples/rvc-app/rvc-common/pics/rvc-app-pics-values

```diff
@@ -45,9 +46,10 @@ RVCOPSTATE.S.M.ERR_MOP_CLEANING_PAD_MISSING=1
 RVCRUNM.S=1
 RVCRUNM.S.A0000=1
 RVCRUNM.S.A0001=1
-RVCRUNM.S.A0002=0
-RVCRUNM.S.A0003=0
-RVCRUNM.S.C01.Tx=1
 RVCRUNM.S.C00.Rsp=1
+RVCRUNM.S.C01.Tx=1
 RVCRUNM.S.F00=0
-RVCRUNM.S.M.CAN_TEST_MODE_FAILURE=1
\ No newline at end of file
+RVCRUNM.S.M.CAN_TEST_MODE_FAILURE=1
+RVCRUNM.S.M.CAN_MANUALLY_CONTROLLED=1
+PIXIT.RVCRUNM.MODE_CHANGE_FAIL=1
```

Comment:
Should these two be removed?

Reply:
If you are referring to the PIXIT codes, they have been removed in [this PR](https://github.com/project-chip/connectedhomeip/pull/31827). I don't think that anything else needs to be removed.